packaging/docker/Dockerfile

```diff
@@ -97,34 +98,43 @@ RUN groupadd --gid 4059 \
             fdb && \
     chown -R fdb:fdb /var/fdb
 
+WORKDIR /tmp
 COPY website /tmp/website/
 
-# Install FoundationDB Binaries
-RUN for file in fdbserver fdbbackup fdbcli fdbmonitor; do \
-        curl --fail -Ls ${FDB_WEBSITE}/${FDB_VERSION}/$file.x86_64 -o $file; \
-        chmod +x $file; \
-        mv $file /usr/bin; \
-    done
+# Install FoundationDB Binaries and additional FoundationDB Client Libraries
+RUN if [ "$TARGETARCH" = "amd64" ]; then \
+        FDB_ARCH=x86_64; \
+    elif [ "$TARGETARCH" = "arm64" ]; then \
+        FDB_ARCH=aarch64; \
+    else \
+        echo "ERROR: unsupported architecture $TARGETARCH" 1>&2; \
+        exit 1; \
+    fi; \
+    CURL="curl --fail -Ls"; \
+    for file in fdbserver fdbbackup fdbcli fdbmonitor; do \
+        URL="${FDB_WEBSITE}/${FDB_VERSION}/$file.$FDB_ARCH"; \
+        echo "Downloading $URL"; \
+        $CURL "$URL" -o "$file" && \
+        chmod +x "$file" && \
+        mv "$file" /usr/bin/ || exit $?; \
+    done && \
+    mkdir -p /var/fdb/lib && \
+    for version in $FDB_LIBRARY_VERSIONS; do \
```

Comment:
I don't think we produce arm binaries for older versions like ( '6.3.25' '7.1.57' '7.3.37') mentioned in the `build-images.sh` script. So by default, the build would fail. Other than that, the PR looks good to me. Maybe at certain point we can remove old versions in the script.